docs/release-notes/13-2-0/README.md

```diff
@@ -3,22 +3,87 @@ title: OpenProject 13.2.0
 
-## Important updates and breaking changes
+We also added the option to **filter the project member list**, allowing project administrators to easily filter through the project member lists based on various roles, groups and shares. Instance administrators can now also allow for users to **change work package status without the rights to edit a work package**. 
 
-<!-- Remove this section if empty, add to it in pull requests linking to tickets and provide information -->
+Furthermore, it is now **possible to set users with whom a work package has been shared as assignees or responsible**. The **quick content menu** in the Gantt view has now shows work package relations. 
 
-<!--more-->
+Finally, **several fields were renamed**: 
+
+- **Estimated time** is now called Work (calculation of **Work** has also been updated)
+- **Remaining hours** is now called Remaining work (calculation of **Remaining work** has also been updated)
+- **Progress (%)** is now called **% Complete**
+
+As always, this release contains several bug fixes and we recommend updating to the newest version.
+
+
+## Manual project folders for OneDrive/SharePoint storages (Enterprise Add-On)
+
+With OpenProject 13.2, manual project folders have become available for OneDrive/SharePoint storage, further improving access to essential project files. 
+
+When project folders are configured, all project-related files are automatically uploaded and organized within the specified folder. To activate project folders, administrators are required to designate the desired folder as the project folder and manually configure the associated permissions. This feature enhances user convenience and file organization within the OpenProject environment.
+
+![Manual project folders for OneDrive/SharePoint storages in OpenProject](onedrive-storage-add-folders.png)
+
+See our user guide to learn more about this Enterprise add-on and how to [use the SharePoint integration](../../user-guide/file-management/one-drive-integration/).
+
+## Show file links of files that are not available to the user in the cloud storage
+
+In 13.2 we improved the functionality of file storages. Even if a file has been deleted on a file storage, it will still be displayed under the Files tab in OpenProject work packages, allowing users to better keep track of project files. 
+
+![Deleted file storage file in OpenProject](deleted-file.png)
+
+This new feature is available for [both file storage integrations](../../user-guide/file-management). 
+
+## Filter for roles, groups, and shares in the project members list
+With OpenProject 13.2, users can filter the list of project member based on user roles, groups, and shared work packages. This enables project members to promptly identify others with specific roles or individuals outside the project team who have  gained access through the newly introduced Sharing feature. 
```

Comment:
double space here:
... have  gained (access through the newly introduced Sharing feature.)